src/js/components/aboutTheData/AboutTheDataPage.jsx

```diff
@@ -64,82 +64,82 @@ const AboutTheDataPage = ({ history }) => {
         });
     };
 
+    const slug = `submission-statistics/${search}`;
+
+    const handleShare = (name) => {
+        handleShareOptionClick(name, slug, getAllAgenciesEmail(getBaseUrl(slug), urlPeriod, activeTab));
```

Comment:
The email body is showing up with an incorrectly formatted URL 
```
View agency submission details on USAspending: https://www.usaspending.gov/submission-statistics/?fy=https%3A%2F%2Fwww.usaspending.gov%2Fsubmission-statistics%2F%3Ftab%3Dsubmissions%26fy%3D2021%26period%3D5&period=5&tab=submissions
```

Reply:
yea everywhere we use the handleShareOptionClick we were passing the slug of the url but we dont need in this case. Replaced this with the `urlFy` as the function expects.